OpenRA.Game/Renderer.cs

```diff
@@ -123,26 +128,47 @@ public void InitializeDepthBuffer(MapGrid mapGrid)
 		public void BeginFrame(int2 scroll, float zoom)
 		{
 			Context.Clear();
+
+			var resolution = Resolution;
```

Comment:
No need to assign this to a new var, we are using it it directly in backingResolution

Reply:
`Resolution` is a property that goes through a couple of abstractions and a layer of locking to return. I'm caching it here because we use it multiple times.

Reply:
Added a comment to clarify this.